### DISCUSSION TITLE
Windows startup priority (battery drainage because forgot startup)

### DISCUSSION THREAD

**Discussion by ugryy** [seerge/g-helper#976] [General]
Firstly, thanks so much for this simple but awesome app. Saved me from armory crate.

Well I have a little problem and a suggestion about the solution I think.
I'm using my rog zephyrus g14 2021 at battery mostly. After I used the laptop, shut down as usual and wake up tomorrow then open my pc. (I have checked "Launch at Windows startup" already) but g-helper doesn't start most of the time at startup. As I don't switch profile often (using silent %90 time) I don't notice it until I realize that my battery drains way faster (15-25mWh). I have g-helper pinned on taskbar, so when I open g-helper immediately battery usage drops to normal (around 9mWh). 
I looked on Task Manager --- Startup  so maybe I can change startup priority but  I cannot see g-helper there. So I thinked about it, I am also using app called Wallpaper Engine. It has the Priority Startup button. After execute it one time, startup was much much before and without issue maybe?  
I wonder if same thing maybe added to g-helper. 
With my best regards.

**Comment by seerge** [comment]
@ugryy G-Herlper task resides under windows Task Manager app, it runs as soon as you log in. Running before that is not possible, as it's app with UI and it requires windows user to run.

By starting itself g-helper doesn't drop your discharge rate anyhow (it does nothing for discharge in general). I can attribute higher discharge rate you see in the beginning cause your computer spends resources to boot, and that metric takes some time to adjust / cool down :)
